frontend/src/types/analytics.ts

```diff
@@ -216,6 +216,18 @@ export type Events = {
     /** The permalink to the creator's profile */
     url: string
   }
+  /**
+   * Description: The user visits a source's link in the single result UI
+   * Questions:
+   *   - Are source links clicked much? Does Openverse increase visibility
+   *     of included sources?
+   */
+  VISIT_SOURCE_LINK: {
+    /** The unique ID of the media */
+    id: string
```

Comment:
Can we rename this to `collectionType`? The comment doesn't appear to be related to this event either.

```suggestion
    /** The type of collection, either "source" or "creator" (tags do not have external links) */
    collectionType: "source" | "creator"
```

Reply:
The `id` property here should be removed, and `collectionType` is not necessary.

This event is fired when the user clicks on the external source link on the source collection page (the black button in the snapshots here). 
When the user clicks on the `creator`'s external link, a different event (`VISIT_CREATOR_LINK`) is fired.

Reply:
We could probably replace the url with the source slug - to make sure that if we ever change the URL we link to, the event is still the same.

Reply:
Good idea!